recipes/opencv/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = "2.4.12" %}
+{% set filename = "opencv2-%s.zip" % version %}
+
+package:
+  name: opencv
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}
+  url: https://github.com/Itseez/opencv/archive/{{ version }}.zip
+  sha1: 2225768a48dbf99426bf77451a433ed432f54d8e
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+
+requirements:
+  build:
+   - cmake
+   - curl
+   - numpy x.x
+   - python
+   - eigen 3.*
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
+
+  run:
+   - python
+   - numpy x.x
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
+
+test:
+  imports:
+    - cv2
+    - cv2.cv
+  commands:
+    # Verify dynamic libraries.
+    {% set opencv_libs = [
+            "core",
+            "flann",
+            "imgproc",
+            "highgui",
+            "features2d",
+            "calib3d",
+            "ml",
+            "video",
+            "legacy",
+            "objdetect",
+            "photo",
+            "gpu",
+            "nonfree",
+            "contrib",
+            "stitching",
+            "superres",
+            "videostab"
+    ] %}
+    {% for each_opencv_lib in opencv_libs %}
+    - test -f $PREFIX/lib/libopencv_{{ each_opencv_lib }}.dylib   # [osx]
+    - test -f $PREFIX/lib/libopencv_{{ each_opencv_lib }}.so      # [linux]
+    {% endfor %}
```

Comment:
Apparently there is no `FindOpenCV.cmake` - this functionality is provided by `$PREFIX/share/OpenCV/OpenCVConfig.cmake` - at least that is what I gather from [here](https://bugs.launchpad.net/ubuntu/+source/opencv/+bug/1289094). This file is currently installed, but it adds this `share` folder. Do we want it there? There doesn't seem to be a standard place to put this stuff? I've also got other packages locally that has installed to places such as '$PREFIX/lib`, '$PREFIX/PACKAGE/lib' and '$PREFIX/cmake`.